docs/UserGuide.md

```diff
@@ -60,38 +60,55 @@ Oops! There seems to be an error, please check the format of your command again.
 
 Expected output (success):
 ```agsl
-Fosterer Jerry Tan is successfully added!
+New fosterer added: Jerry Tan; Phone: 98765412; Email: jerry123@example.com; Address: Baker street, block 5, #27-01; Housing: HDB; Availability: NotAvailable; Animal name: Dexter; Animal type: current.Cat; Tags: [Urgent]
 ```
 
 Expected output (fail):
-```agsl
-Oops! There seems to be an error, please check the format of your command again.
-```
+
+| Scenario                                                                                                               | Error Message                                            |
+|------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| Without `n/`, `p/`,`e/`, `a/`                                                                                          | `Invalid command format! add: Adds a person to the address book. Parameters: n/NAME p/PHONE e/EMAIL a/ADDRESS housing/HOUSING availability/AVAILABILITY animal/ANIMAL_NAME animalType/ANIMAL_TYPE [t/TAG]... Example: add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 housing/HDB availability/NotAvailable animal/Dexter animalType/current.Dog t/Urgent t/goodWithDogs`                                                                  |
+| Without `housing/`                                                                                                     | `Housing is required. Please indicate as 'nil' if information is not available.` |
+| Without `availability/`                                                                                                | `Availability is required. Please indicate as 'nil' if information is not available.` |
+| Without `animal/`                                                                                                      | `Animal name is required. Please indicate as 'nil' if information is not available.` |
+| Without `animalType/`                                                                                                  | `Animal type is required. Please indicate as 'nil' if information is not available.` |
+| `availability/nil` and `animal/` not 'nil'                                                                             | `Animal name should be 'nil' when availability is 'nil'.` |
+| `availability/nil` and `animalType/` not 'nil'                                                                         | `Animal type should be 'nil' when availability is 'nil'.` |
+| `availability/Available` and `animal/` not 'nil'                                                                       | `Availability cannot be 'Available' when an animal name is provided; animal name should be 'nil'.` |
```

Comment:
I think these can be phrased more intuitively: avail/nil **but**  animal/ is not nil

Reply:
Changed!